lib/Dialect/Handshake/HandshakeTypes.cpp

```diff
@@ -99,9 +103,152 @@ checkChannelExtra(function_ref<InFlightDiagnostic()> emitError,
   return success();
 }
 
+/// Parse the extra signals from the IR representation.
```

Comment:
As much as possible please wrap comment correctly at 80 characters. Newlines are not honored by most IDEs when displaying a function's comment on hover.

Reply:
I want to keep the newlines as much as possible for readers viewing this comment directly in the file.
I mostly use VSCode (with clangd), which seems to respect the newlines:
![image](https://github.com/user-attachments/assets/2fd66b73-ff63-4442-9f98-565e23a6dc2b)

Reply:
You're right, I just checked and now it seems to honor new lines. I could swear it used to not do that...